### DISCUSSION TITLE
raspberrypi sdimg missing?

### DISCUSSION THREAD

**Discussion by tsia** [Igalia/meta-webkit#283] [Q&A] [answered]
i tried to build WPE for the raspberry pi 3. i followed the instructions on https://github.com/Igalia/meta-webkit/wiki/RPi (Option 1). Once i gave the build host enough ram the build went through without any errors however according to the wiki there should be a `*.rpi-sdimg` but in the mentioned folder i only have a `*.rootfs.ext3`. am i missing something?

Thanks

**Comment by suniltop** [comment, marked as answer]
meta-raspberrypi layer supports generating sdcard image.
Only you have to add lines that is given below to conf/local.conf and then build again

IMAGE_FSTYPES = "tar.xz ext3 rpi-sdimg"
